src/pipelines.py

```diff
@@ -345,117 +226,79 @@ def mask_postprocessing(model, config, save_output=False):
     return nuclei_filter
 
 
-def contour_postprocessing(model, config, save_output=True):
-    contour_resize = Step(name='contour_resize',
-                          transformer=make_apply_transformer(resize_image,
-                                                             output_name='resized_images',
-                                                             apply_on=['images', 'target_sizes']),
-                          input_data=['input'],
-                          input_steps=[model],
-                          adapter={'images': ([(model.name, 'contour_prediction')]),
-                                   'target_sizes': ([('input', 'target_sizes')]),
-                                   },
-                          cache_dirpath=config.env.cache_dirpath,
-                          save_output=save_output)
+def mask_postprocessing_simple(model, config, save_output=False):
```

Comment:
@taraspiotr I don't like the `_simple` suffix (though I am culprit of using it too). It is to generic. Why not `_masks` ?